src/activity_handlers.cpp

```diff
@@ -1028,26 +1028,26 @@ void activity_handlers::reload_finish( player_activity *act, player *p )
         return;
     }
 
+    std::string msg = _( "You reload the %s." );
```

Comment:
I don't know if `RELOAD_ONE` means anything when not a gun, but you could avoid reassigning `msg` like this:

```c++
std::string msg;
if( reloadable->has_flag( "RELOAD_ONE" ) ) {
    ...
} else {
    msg = _( "You reload the %s." );
}

```